affinidi-messaging-mediator/src/database/list.rs

```diff
@@ -43,6 +43,8 @@ impl DatabaseHandler {
                 .arg(&key)
                 .arg(start)
                 .arg(end)
+                .arg("COUNT")
+                .arg(100)
```

Comment:
WDYT to move this magic number to the header of the file and make a constant, for example, MAX_MESSAGES_LIMIT or something similar.